torch/_functorch/eager_transforms.py

```diff
@@ -364,7 +364,7 @@ def error_if_complex(func_name, args, is_input):
             raise RuntimeError(err_msg)
 
 @exposed_in("torch.func")
-def jacrev(func: Callable, argnums: Union[int, Tuple[int]] = 0, *, has_aux=False,
+def jacrev(func: Callable, argnums: Optional[argnums_t] = 0, *, has_aux=False,
```

Comment:
I wrote this on the issue, but argnums cannot actually be None, it must be int or Tuple[int, ...]. If we accept `None` here then that's a bug (and we should add a check against it)

Reply:
Ok my bad, I'll fix this. So all in all we need to change the type hint and to make a check in both to verify against `None`, I'll delete the test too.

Reply:
yup, thank you

Reply:
I think, the new version matches our discussions.